hoomd/md/test/test_constraint_sphere.cc

```diff
@@ -1,140 +0,0 @@
-// Copyright (c) 2009-2021 The Regents of the University of Michigan
```

Comment:
Why did this file need to be removed?

Reply:
Tests failed for some reason (they did not on other tests files modified and for economy of time since this class is slotted for removal not porting, I just deleted it.